ce/ce/installers/util.ts

```diff
@@ -16,9 +16,17 @@ export function artifactFileName(name: string, install: Installer, extension: st
     result += '-';
     result += install.lang;
   }
+  // add the version number into the filename too.
+  result += '-' + (version || '');
```

Comment:
Isn't the `||` dead because it isn't `string?`?

Reply:
Yeah, that seems a smidge redundant. fixing